jsonnet/templates.libsonnet

```diff
@@ -119,9 +127,10 @@
         },
         alert: {
           name: 'NodesHealthLow',
-          message: '"{{ $labels.node }}": Node Health Low {{ $value }}%',
+          message: 'Nodes Health Low {{ $value }}%',
```

Comment:
Why `$labels.node` was removed from alert message here?

Reply:
It was useless in my opinion. [Expresion](https://github.com/dNationCloud/kubernetes-monitoring/blob/main/jsonnet/templates.libsonnet#L108) uses sums and always return only 1 number, therefore applying `avg` afterwards is useless.

Reply:
Agree